docs/troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+# Troubleshooting Guide
+
+## Index
+
+- [Troubleshooting AKS cluster creation](#troubleshooting-aks-cluster-creation)
+    + [Service principal clientID not found](#service-principal-clientid-not-found)
+- [Troubleshooting BKPR installation](#troubleshooting-bkpr-installation)
+    - [Object with the same value for property exists](#object-with-the-same-value-for-property-exists)
+
+## Troubleshooting AKS cluster creation
+
+### Service principal clientID not found
+
+If you notice the following error message from `az aks create`, it could indicate the Azure authentication token has expired.
+
+    ```
```

Comment:
When I view this file in GitHub, the triple back ticks get rendered if they are indented. Perhaps you should remove the indentation?

Reply:
whoa.. i didn't even notice the rendering issue

Reply:
fixed!